src/Utils/AttributesUtils.php

```diff
@@ -0,0 +1,145 @@
+<?php
+namespace Automattic\WooCommerce\Blocks\Utils;
+
+/**
+ * AttributesUtils class used for getting class and style from attributes.
+ */
+class AttributesUtils {
+
+
+	/**
+	 * Get class and style for font-size from attributes.
+	 *
+	 * @param array $attributes Block attributes.
+	 *
+	 * @return string
+	 */
+	public static function get_font_size_class_and_style( $attributes ) {
+
+		$font_size        = $attributes['fontSize'];
+		$custom_font_size = $attributes['style']['typography']['fontSize'];
```

Comment:
If you set `define( 'WP_DEBUG', 'True' );` in your site's `wp-config.php`, you will probably see errors appear on the screen because of this line. That's because `typography` might be missing inside the `style` object.

I think you will need to use `array_key_exists()` (or something similar?) every time you try to access an attribute inside `$attributes['style']`.